### PR TITLE
Added reports from Ralph 2

### PR DESCRIPTION
**Pull request by szok**
- Added asset relation report
- Added licence relation report

### CODE REVIEW

**Comment by vi4m** [comment]
Nice html explanation needed, like in present reports.

File: src/ralph/reports/views.py [hunk 1]
```diff
@@ -173,6 +232,106 @@ def prepare(self, model, dc=None):
             )
 
 
+class BaseRelationsReport(BaseReport):
+    template_name = 'reports/report_relations.html'
+    with_modes = True
+    links = False
+
+    def prepare(self, model, *args, **kwargs):
```
Comment (mkurek) [inline src/ralph/reports/views.py]: do we need two functions for this: `prepare` and `get_result`?

File: src/ralph/reports/views.py [hunk 2]
```diff
@@ -173,6 +232,106 @@ def prepare(self, model, dc=None):
...
+        pass
+
+    def get_response(self, request, result):
```
Comment (mkurek) [inline src/ralph/reports/views.py]: what do you think about extracting it to separated class (`CSVReportMixin`)?

File: src/ralph/reports/views.py [hunk 3]
```diff
@@ -173,6 +232,106 @@ def prepare(self, model, dc=None):
...
+    slug = 'asset-relations'
+    name = _('Asset - relations')
+    description = _('Asset list of information about the user, owner, model.')
+    filename = 'asset_relations.csv'
+    headers = ASSETS_COLUMNS
+
+    def get_assets_rows(self, model):
+        queryset = model.objects.all()
+        for asset in queryset:
+            row = [getattr_dunder(asset, column) for column in ASSETS_COLUMNS]
```
Comment (mkurek) [inline src/ralph/reports/views.py]: how many SQLs are executed during generating this report? wouldn't it be better to pass these fields to `values_list` and do everything on SQL level?

File: src/ralph/reports/views.py [hunk 4]
```diff
@@ -173,6 +232,106 @@ def prepare(self, model, dc=None):
...
+
+
+class AssetRelationsReport(BaseRelationsReport):
+    slug = 'asset-relations'
+    name = _('Asset - relations')
+    description = _('Asset list of information about the user, owner, model.')
+    filename = 'asset_relations.csv'
+    headers = ASSETS_COLUMNS
+
+    def get_assets_rows(self, model):
```
Comment (mkurek) [inline src/ralph/reports/views.py]: there should be no option to generate report for ALL assets - it's useless; just maintain two separated reports for BO and DC and create separated columns for BO and DC.

File: src/ralph/reports/views.py (hunk 2 above)
Comment (szok) [inline src/ralph/reports/views.py]: :+1:

**Comment by mkurek** [comment]
:ok:

File: src/ralph/reports/views.py [hunk 5]
```diff
@@ -26,6 +31,33 @@ def get_choice_name(choices_class, key, default='------'):
         return 'Does not exist for key {}'.format(key)
 
 
+class CSVReportMixin(object):
+    """CSV report mixxin.
```
Comment (ar4s) [inline src/ralph/reports/views.py]: mixxin -> mixin

**Comment by ar4s** [comment]
:ok:

**Pull request merged**